.travis.yml

```diff
@@ -19,11 +19,10 @@ install:
   - conda config --add channels defaults
   - conda config --add channels conda-forge
   - conda config --add channels bioconda
-  - conda install -c bioconda -c conda-forge snakemake
-  - conda create -q -n snakemake snakemake>=5.3.1 python=3.6
+  - conda create -q -n snakemake 'snakemake>=5.3.1' 'python>=3.10'
```

Comment:
Requires python >= 3.10 to get the ` | None` working.

Reply:
I believe this is only for documentation updates, not the code itself.

Reply:
Aaah. But then, which python is used? It was written `python3.6 setup.py install`...

Reply:
I updated it on the branch to 3.11